### DISCUSSION TITLE
Is it possible to change how abbreviations for months are written?

### DISCUSSION THREAD

**Discussion by sokulski** [js-fns/date-fns#2429] [Q&A]
Many style guides and date standards stipulate that when months are abbreviated to 3 letters that June and July specifically not be abbreviated.

I think this is even true in the Unicode Standard, which posts this example:
`EEE, MMM d, ''yy | Wed, July 10, '96`

Is it possible to get this behavior out of date-fns as it stands today?

**Comment by tan75** [comment]
hi @sokulski 
could you please add more information as of what locale is not correct when it comes to months formatting? 
When creating a new locale, we ask date-fns contributors to refer to [CLDR](https://unicode.org/cldr/cldr-aux/charts/32/summary/root.html) - **months abbreviated** section.
Please let us know which locale is out of sync and we will fix it or feel free to submit a PR if you wish to contribute.

**Comment by sokulski** [comment]
Hi @tan75,

Thanks for taking the time to look at this, and for linking the CLDR.

The behavior of date-fns mirrors the month abbreviated section perfectly.

My confusion stemmed, I think, from reference to the Unicode Standard #35, section 4, which is referenced in the date-fns docs.

In the standards, specifically the section pertaining to [Date Format Patterns](https://unicode.org/reports/tr35/tr35-dates.html#8-date-format-patterns), there is this example:

`EEE, MMM d, ''yy | Wed, July 10, '96`

I took that to mean that July should not be abbreviated to Jul when using the token MMM. But date-fns format function does abbreviate the month.

I don't know that there is a technical problem here, but instead maybe my request is a bit odd?

My end goal is to avoid abbreviated short months, as is a somewhat common request in US English style guides. Is there a way to achieve that?
